spec/services/protocol_importers/import_protocol_service_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe ProtocolImporters::ImportProtocolService do
+  let(:user) { create :user }
+  let(:team) { create :team }
+  let(:protocol_params) { attributes_for :protocol, :in_public_repository }
```

Comment:
Did you mean `in_repository_public` here?

Reply:
It's ok. It is the name of trait in the factory.